Init (#1)
CU-DBMI/wall-of-software: README.md

> # CU DBMI Software Registry

Reviewer: (aligning with other content)
```suggestion
# CU DBMI Wall of Software
```

> # CU DBMI Software Registry

Reviewer: Naming things is a challenge! Openly wondering if another name would provide additional elegance or sense of pride in the work for the department. Mostly I'm imagining members of DBMI saying something like "Hey I just got added to the \<name goes here\> today!" (what would add excitement / enhancement to this?). To this point, I think moving from "Software Registry" to "Wall of Software" is a great change (getting "registered" isn't often exciting and might be a negative).

Other things that come to mind: "DBMI Software Tapestry", "Software Hall of Fame", "Software Achievement Wall".  I can also see how being more humble / frank about it would be better 🙂 (feel free to dismiss this as overly-semantic).

Author: Yeah I think WOS is more descriptive and more fun. Tapestry would also be cool. Wouldn't wan't to call it HOF just b/c if you're package isn't in there it may feel bad.

> The [Department of Biomedical Informatics at the University of Colorado](https://medschool.cuanschutz.edu/dbmi) investigates complex problems in medicine and biology using integrated computational technology.

Reviewer: ```suggestion
The [Department of Biomedical Informatics (DBMI) at the University of Colorado](https://medschool.cuanschutz.edu/dbmi) investigates complex problems in medicine and biology using integrated computational technology.
```

This repository and website contains a non-comprehensive list of tools, packages, workflows, resources, and other software that have been developed by our diverse group researchers, scientists, developers, and collaborators in support of [our mission](https://medschool.cuanschutz.edu/dbmi/about-us).

> [⭐️⭐️ View the List ⭐️⭐️](https://CU-DBMI.github.io/software-registry)

Reviewer: ```suggestion
[⭐️⭐️ View the List ⭐️⭐️](https://CU-DBMI.github.io/wall-of-software)
```

## Submit Your Software

> If you work for or with the department, you are eligible to submit your software here.

Reviewer: Would it make sense to provide a note here that the Software Engineering Team could help with this if someone would like their work to appear here? Mostly I'm wondering this out of kindness for folks who may not understand the technology behind it (might be working in other capacities).

> To start, [open a pull request in this repo](https://github.com/CU-DBMI/software-registry/pulls) with the following changes:

Reviewer: ```suggestion
To start, [open a pull request in this repo](https://github.com/CU-DBMI/wall-of-software/pulls) with the following changes:
```

- Add an _entry_ to **the top** of `registry.json`.
  - A short ~3-20 char name.
  - A short ~10-30 char description.
  - A link to a website or repo.
- Add a _web-friendly_ image to `/images`.

>   - Filename **must** be lower-case, kebab-case version of entry name.

Reviewer: Consider adding in a link to describe kebab-case.
```suggestion
  - Filename **must** be lower-case, [kebab-case](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case) version of entry name.
```

  - **Must** have extension `.jpg`.
  - Roughly square.
  - Enough margins for main content to fit nicely inside hexagon.

>   - Larger than ~400×400, smaller than ~1000×1000

Reviewer: ```suggestion
  - Larger than ~400×400, smaller than ~1000×1000 pixels
```

  - Better than ~80% jpeg compression level, smaller than ~50 kB.

Shortly after opening a PR, a link should appear to a preview of the site with your changes.

Entries are shuffled randomly on each page visit to not give preferential treatment!

### Printed

> If you'd also like your image to be printed out and posted on the wall of the DBMI office (7th floor Anschutz Health Sciences Building), **also** include these changes:

Reviewer: ```suggestion
If you'd also like your image to be printed out and posted on the wall of the DBMI office (6th floor Anschutz Health Sciences Building), **also** include these changes:
```
